develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown

```diff
@@ -268,6 +237,9 @@ build your portlet URL:
       return portletURL;
     }
 
+Now that the portlet URL is updated, you can update the search container to 
+reflect the changes.
+
 ## Including the Selected Column and Sorting Order in the Search Container [](id=including-the-selected-column-and-sorting-order-in-the-search-container)
```

Comment:
The same as commented above, I think this is not relevant for a management toolbar implementation.